pkg/cgroups/cgroups.go

```diff
@@ -49,17 +49,23 @@ func (subsys Subsystems) SetCPU(name CgroupID, cpus int) error {
 		return UnsupportedError("cpu")
 	}
 
-	quota := cpus * CPUPeriod
-	if cpus == 0 {
-		// setting -1 here will unrestrict the cgroup, so the period won't matter
-		quota = -1
```

Comment:
So we were setting it to negative 1 in the code but not actually writing it to the file system? Oops.

Reply:
no, we were writing to the file system as -1 for quota if there was no cgroup for cpu